### PR TITLE
DTSPO-6900.  Using old ZAPproxy image

### PR DESCRIPTION
**Pull request by mokainos**
https://github.com/zaproxy/zaproxy/issues/7111

### CODE REVIEW

**Review by timja — APPROVED** [review]
this is okay for now to unblock if you've tested it in a pipeline but we need a solution so we aren't stuck here forever

**Comment by thc202** [comment]
There's an update in the referenced issue.

**Review by timja — CHANGES_REQUESTED** [review]
let's hold off on this

**Pull request closed without merging**
